recipes/prometheus-cpp/all/conanfile.py

```diff
@@ -83,6 +83,7 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "prometheus-cpp"
```

Comment:
Missing come details on the components

```
[HOOK - conan-center.py] post_package(): ERROR: [PC-FILES (KB-H020)] The conan-center repository doesn't allow the packages to contain `pc` files. The packages have to be located using generators and the declared `cpp_info` information (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H020) 
[HOOK - conan-center.py] post_package(): ERROR: [PC-FILES (KB-H020)] Found files: ./lib/pkgconfig/prometheus-cpp-core.pc; ./lib/pkgconfig/prometheus-cpp-pull.pc; ./lib/pkgconfig/prometheus-cpp-push.pc (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H020) 
ERROR: [HOOK - conan-center.py] post_package(): Some checks failed running the hook, check the output
```

At a minimum please add a fix me

```suggestion
        # FIXME: Missing prometheus-cpp-core.pc, prometheus-cpp-pull.pc prometheus-cpp-push.pc
        self.cpp_info.names["cmake_find_package"] = "prometheus-cpp"
```

You can easily add these in

Example after L93

```py
self.cpp_info.components["prometheus-cpp-core"].names["pkg_config"] = "prometheus-cpp-core"
```